### PR TITLE
Record greg2

### PR DESCRIPTION
**Pull request by gerghan**
Changed page titles to specific ones. Added back button to Record page

### CODE REVIEW

File: app/src/main/java/com/example/justi/markthat/Record.java [hunk 1]
```diff
@@ -1,13 +1,36 @@
...
+        android.support.v7.app.ActionBar actionBar = getSupportActionBar();
+        if (actionBar != null) {
+            actionBar.setDisplayHomeAsUpEnabled(true);
+            actionBar.setTitle(recordTitle);
+        }
+        //alt method that dont work
+        //ActionBar actionBar = getActionBar();
+        //actionBar.setDisplayHomeAsUpEnabled(true);
+        //change title of page
+       // setTitle(recordTitle);
```
Comment (jstnzl) [inline app/src/main/java/com/example/justi/markthat/Record.java]: Remove the comments if they dont work or arent needed

File: app/src/main/java/com/example/justi/markthat/ViewRecording.java [hunk 2]
```diff
@@ -9,5 +9,7 @@
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_view_recording);
+        //title
+        setTitle("MarkThat - View Recording(s)");
```
Comment (jstnzl) [inline app/src/main/java/com/example/justi/markthat/ViewRecording.java:13]: Change home to MarkThat - Home to be consistent

**Review by jstnzl — CHANGES_REQUESTED** [review]
Back button doesnt actually work, it just loads the previous page on top.
steps to repo- go to record, use software back button, hit hardware back button
I guess it's fine if we dont have back button for now so if you cant figure it out let me know we'll find something else to do

**Comment by gerghan** [comment]
Deleted lines for back button and changed title of home page

**Pull request merged**